cli/medperf/entities/schemas.py

```diff
@@ -74,7 +74,7 @@ class Config:
         use_enum_values = True
 
 
-class MedperfSchema(MedperfBaseSchema):
+class MedperfSchema(BaseModel):
```

Comment:
Now the code works as `MedperfSchema` is always used together with `Entity`. However, don't it look strange that `MedperfSchema` is not connected with `MedperfBaseSchema` anymore? Maybe we can refactor it a bit more?
As far as I see, the key is `TestReport` that should inherit from `MedperfBaseSchema` but not have fields from `MedperfSchema` (?) I don't see a good solution here at glance, but I believe you thought about it, and all possible solutions of joining everything together have their own key disadvantages, right?

Reply:
you are right, it now looks strange. I think I have an idea, will push a change soon